File: Clocky Mc Clock-Face.py
import math
def what_time_is_it(angle):
    f,i = math.modf(angle/30)
    i = 12 if i==0 else i
    return f"{str(int(i)).zfill(2)}:{str(math.floor(f*60)).zfill(2)}"

File: Clocky Mc Clock-Face/Clocky Mc Clock-Face.js
const whatTimeIsIt = (angle) => {
  let hr = Math.floor(angle / 30);
  let mn = Math.floor((angle % 30) * 2);
  if (hr === 0) {
      hr = 12;
  }
  return hr.toString().padStart(2, '0') + ':' + mn.toString().padStart(2, '0');
}
